requirements/devel.txt

```diff
@@ -1,17 +1,4 @@
-# install all mandatory dependencies
--r ../requirements.txt
-
-# install all extra dependencies for full package testing
--r ./extra.txt
-
-# install all loggers for full package testing
--r ./loggers.txt
+-r ./devel-base.txt
```

Comment:
If this file now only has these 2 things, should we remove it and rename `devel-base` to `devel`?